skill/texts.py

```diff
@@ -0,0 +1,43 @@
+import random
+import pymorphy2
+
+morph = pymorphy2.MorphAnalyzer()
+
+
+def __inflect(word, case):
+    return " ".join([morph.parse(x)[-1].inflect(case).word for x in word.split(" ")])
+
+
+def hello():
+    text = "Привет! Это Цифровой дневник"
+    tts = "Привет! Это Цифровой дневник"
+    return text, tts
+
+
+def mistake():
+    text = """Прошу прощения. Во время игры возникла непредвиденная ошибка.
```

Comment:
Тут можно заменить "Во время игры" на "В навыке"
```suggestion
    text = """Прошу прощения. В навыке возникла непредвиденная ошибка.
```